internal/k8s/ako_init.go

```diff
@@ -290,6 +294,12 @@ func (c *AviController) InitController(informers K8sinformers, registeredInforme
 		} else {
 			utils.AviLog.Warnf("Full sync interval set to 0, will not run full sync")
 		}
+		ctrlAuthtoken := os.Getenv("CTRL_AUTHTOKEN")
+		if ctrlAuthtoken != "" {
+			worker = utils.NewFullSyncThread(time.Duration(refreshAuthtokenInterval) * time.Hour)
```

Comment:
this would end up overwriting worker variable which would create problem. 
Lets evaluate if we need a new full sync worker and then use the functionalities, e.g. gracefully shutting down the threads.